pom.xml

```diff
@@ -720,58 +720,58 @@
 
     <!-- Define all plugin versions as properties so individual hierarchies can easily override -->
```

Comment:
Would really like to see all this mess of properties just getting stripped out completely. There are only reason to use properties for version numbers:

1. You need to synchronize a dependency version with a plugin version
2. You have a need to run a CLI execution with another version.

The second case only applies for the hpi and stapler plugins because there is often a need to use `hpi:run` with a different version of jenkins to development test a plugin against newer versions. For all other plugins we should just in-line all the properties and let child plugins use `pluginManagement` if they need a newer version.

Reply:
@stephenc I am going to file this as a followup PR. I'd like to try and keep isolated changes as much as possible, easier to review here. And the "inlining PR" will also hence be only containing this.

Reply:
Well the other thing I would recommend is removing (non-cli) plugins that are not actually used by jenkins core or a bare plugin, eg cargo or gwt seem likely candidates for removal